lib/collector/remote-method.js

```diff
@@ -29,6 +29,11 @@ const USER_AGENT_FORMAT = 'NewRelic-NodeAgent/%s (nodejs %s %s-%s)'
 const ENCODING_HEADER = 'CONTENT-ENCODING'
 const DEFAULT_ENCODING = 'identity'
 
+/**
```

Comment:
did you just start addressing warnings? I don't see these as public facing but maybe I'm not understanding the `public-docs` npm script

Reply:
I think these were automatically added by mistake, possibly by husky. My apologies -- I'll revert these changes.

Reply:
that's fine.  if husky does it then we can't avoid it. I'm wondering which rule is auto fixing to an empty multiline comment stanza

Reply:
Changes to `lib/` reverted :)